docs/proposals/volumes-proposal.md

```diff
@@ -0,0 +1,60 @@
+# Storage/Volume for opencompose
+
+
+```yaml
+services:
+- name: db
+  containers:
+  - image: mysql
+    volumes:
+    - name: db
+      path: /var/lib/mysql
+
+- name: backup
+  containers:
+  - image: backup
+    volumes:
+    - name: db
+      path: /app/store
+      access: ro
```

Comment:
This is binary options, it can be only `ro` or `rw`. How about changing it to something that reflects this more?
For example `readOnly` that can boolean, and if not set that false.